src/display/display_utils.js

```diff
@@ -67,14 +67,23 @@ class FilterFactory {
 
   get #defs() {
     if (!this.#_defs) {
+      const div = this.#document.createElement("div");
+      div.setAttribute("id", "svgDefinitions");
```

Comment:
With the CSS now being inlined, I don't believe that we need an `id` anymore and this line can probably just be removed.